spark/src/test/scala/org/apache/spark/sql/delta/schema/SchemaUtilsSuite.scala

```diff
@@ -2434,8 +2434,15 @@ class SchemaUtilsSuite extends QueryTest
     val base1 = ArrayType(new StructType().add("a", IntegerType))
     val update1 = ArrayType(new StructType().add("b", IntegerType))
 
-    assert(mergeDataTypes(
-      base1, update1, false, false, false, false, allowOverride = false) ===
+    assert(
+      mergeDataTypes(
+        current = base1,
+        update = update1,
+        allowImplicitConversions = false,
+        keepExistingType = false,
+        shouldWidenType = (_, _) => false,
+        caseSensitive = false,
+        allowOverride = false) ===
```

Comment:
Might I suggest putting the result mergeDataTypes into a val first? By the time we get to === I have almost forgotten this is in an assert :P

Reply:
Updated